populate/src/main/resources/seed/portals/ourhealth/siteContent/en/aboutUs.json

```diff
@@ -75,22 +75,26 @@
             "photoBios": [{
               "name": "Pradeep Natarajan",
               "title": "MD MMSc",
-              "blurb": "Co-Principal Investigator, OurHealth\n\nDirector of Preventive Cardiology, Massachusetts General Hospital\n\nPaul & Phyllis Fireman Endowed Chair in Vascular Medicine, Massachusetts General Hospital\n\nAssociate Professor of Medicine, Harvard Medical School\n\nAssociate Member, Broad Institute of Harvard and MIT",
+              "blurb": "Co-Principal Investigator, OurHealth",
+              "detail": "Director of Preventive Cardiology, Massachusetts General Hospital\n\nPaul & Phyllis Fireman Endowed Chair in Vascular Medicine, Massachusetts General Hospital\n\nAssociate Professor of Medicine, Harvard Medical School\n\nAssociate Member, Broad Institute of Harvard and MIT",
```

Comment:
interesting!
First time observed "Broad Institute of Harvard and MIT" , its been "Broad Institute of MIT and Harvard" so far.

Reply:
interesting.. that's text he wrote himself, so I guess he likes Harvard :)